src/main/java/hudson/plugins/clover/CloverBuildAction.java

```diff
@@ -149,10 +156,20 @@ static CloverBuildAction getPreviousResult(Run<?, ?> start) {
         if (c != null) {
             c.setOwner(build);
         }
+        if( this.projectActions == null )
```

Comment:
what about creating a getter:

```
getProjectActions() {
if (projectActions == null) {
  projectActions = new ArrayList<CloverProjectAction>();
}
return projectActions;
}
```

and then using it instead of "this.projectActons", e.g.:

```this.projectActions.add(new CloverProjectAction(build.getParent())); ```

would become

```getProjectActions().add(new CloverProjectAction(build.getParent())); ```